extensions/amp-form/0.1/amp-form.js

```diff
@@ -231,6 +231,26 @@ export class AmpForm {
   }
 
   /**
+   * Triggers 'amp-form-submit' event in 'amp-analytics' and
+   * generates variables for form fields to be accessible in analytics
+   *
+   * @private
+   */
+  triggerFormSubmitInAnalytics_() {
+    const formDataForAnalytics = {};
```

Comment:
This brakes the tests. `formDataForAnalytics` is passed to `this.analyticsEvent_` and in the tests I use `to.be.calledWith(..., expectedFormData)` which calls `hasOwnProperty` to compare the objects.